test/unit/app/utils/ussd/kenya/test_exchange.py

```diff
@@ -0,0 +1,58 @@
+import pytest
```

Comment:
i've split out the state machine tests into different groupings, the one file was getting too ridiculously massive and they're not all that related. on the upside, it's more obvious now if we're missing any tests related to a particular grouping of states. on the downside, pin overlaps a bit with the other groupings, we could end up in a place where we make a mistake thinking a test is in one file and not doing it in another...? not that it's that much easier to spot it in one massive file

Reply:
I think this is much better! saves scrolling for a billion years trying to find things.